doc/release-notes/0.15/0.15.md

```diff
@@ -0,0 +1,190 @@
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/54">#54</a></td>
+<td valign="top">OpenJ9 now supports all POSIX signals listed in <a href="https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/test/jdk/sun/misc/SunMiscSignalTest.java"><tt>SunMiscSignalTest.java</tt></a>.</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">Earlier release notes reported gaps in signal handling support, which are now resolved in this release. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/5239">#5293</a></td>
+<td valign="top">Compatibility support for <tt>-XX:OnOutOfMemoryError</tt></td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">For compatibility with HotSpot, this option can now be used to run a command or list of commands when a
+<tt>java.lang.OutOfMemoryError</tt> exception occurs. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/6013">#6013</a></td>
+<td valign="top">Support for Transparent HugePage allocation</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">The VM now supports Transparent HugePage on Linux when you use the <tt>madvise</tt> (<tt>/sys/kernel/mm/transparent_hugepage/enabled</tt>) setting. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/5538">#5538</a></td>
+<td valign="top">New <tt>jmap</tt> tool</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">For compatibility, OpenJ9 includes an independent implementation of the <tt>jmap</tt> tool, which
+prints statistics about classes on the heap, including number of objects and aggregate size.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/2679">#2679</a></td>
+<td valign="top">JEP 331 is implemented</td>
+<td valign="top">OpenJDK11 and later</td>
+<td valign="top">An implementation of <a href="http://openjdk.java.net/jeps/331">JEP 331</a> (Low-Overhead Heap Profiling) is now available. Restrictions are detailed in the <a href="https://eclipse.github.io/openj9-docs/version0.15/">user documentation</a>. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/5882">#5882</a></td>
+<td valign="top"><tt>-Xdiagnosticscollector</tt> option removed</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">No impact. Redundant option removed.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/5374">#5374</a></td>
+<td valign="top">New heuristics to implement compaction during idle garbage collection.</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">Heuristics are added to automatically compact the heap during idle garbage collection (GC) processing. Therefore, if
+<tt>-XXIdleTuningGCOnIdle</tt> is enabled, compaction might occur. To force compaction every time idle GC operations take place,
```

Comment:
The options are `-XX:+IdleTuningGCOnIdle` and `-XX:+IdleTuningCompactOnIdle`

Reply:
Fixed